lighthouse-core/audits/speed-index-metric.js

```diff
@@ -48,16 +48,16 @@ class SpeedIndexMetric extends Audit {
    * @return {!Promise<!AuditResult>} The score from the audit, ranging from 0-100.
    */
   static audit(artifacts) {
-    const trace = artifacts.traces[this.DEFAULT_PASS];
-    if (typeof trace === 'undefined') {
-      return SpeedIndexMetric.generateAuditResult({
-        rawValue: -1,
-        debugString: 'No trace found to generate screenshots'
-      });
-    }
+    const trace = artifacts.traces[Audit.DEFAULT_PASS];
 
     // run speedline
-    return artifacts.requestSpeedline(trace).then(speedline => {
+    return Promise.resolve().then(_ => {
```

Comment:
Is this first promise something we can cleanup between audits? There are a few of them now